### ISSUE TITLE
Godot4 support?

### ISSUE THREAD

**Issue by GeorgeS2019** [SpookyCorgi/phiz#5]
https://www.youtube.com/watch?v=XduuHNOGGqI

Would you consider supporting Godot4 as first class as Unity and Unreal. 

This would be BIG contribution to Godot community.

**Comment by SpookyCorgi** [comment]
Yes Godot is definitely on the priority list! Do you have resources for how people usually pilot avatars or receive data in Godot. For example embedded websocket/webrtc or OSC? Also what's the most common animation system in Godot?

**Comment by GeorgeS2019** [comment]
Thx so much...will reply soon...

#FYI
- https://github.com/NVIDIA/MAXINE-AR-SDK
- https://github.com/google/mediapipe/issues/4210#issuecomment-1501618356

**Comment by GeorgeS2019** [comment]
@SpookyCorgi  I hope others could come to support you. There are many relevant [demo ](https://github.com/gdquest-demos/godot-4.0-new-features/issues/34)in this repos

**Comment by GeorgeS2019** [comment]
https://github.com/ButzYung/SystemAnimatorOnline/issues/7#issue-1712711078

**Comment by GeorgeS2019** [comment]
@SpookyCorgi 
Here is the latest [Ready Player Me for Godot](https://github.com/srcnalt/ReadyPlayerMe-Godot-Test/issues/1#issue-1713856035)
